hack/generate/gen-test-framework.sh

```diff
@@ -8,23 +8,22 @@ set -ex
 
 source hack/lib/test_lib.sh
 
-# Define vars
-DOCKERFILE="build/Dockerfile"
-
 # Go inside of the mock data test project
 cd test/test-framework
 
-# Create the required files in the mock test/test-framework just to allow run from it the oprator-sdk commands
-echo > $DOCKERFILE
-go mod init test-framework
-
-# Remove files that are required and created just to exec the commands.
-trap_add 'rm -rf $DOCKERFILE go.mod go.sum' EXIT
+# Ensure test-framework is up-to-date with current Go project dependencies.
+# NOTE: the SDK dependency version still needs updating on a new release.
+sdk_version="$(go list -m -f '{{.Version}}' github.com/operator-framework/operator-sdk)"
```

Comment:
TIL about this command! If there's equivalent go code, maybe in a future PR, we should consider outputting this version in the `go.mod` scaffold directly. Though maybe it's moot with the kubebuilder transition.